templates/global/quantity-input.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see     https://docs.woocommerce.com/document/template-structure/
  * @package WooCommerce/Templates
- * @version 3.6.0
+ * @version 3.9.1
```

Comment:
This should be 3.10 since we do not roll out enhancement with fix releases anymore, we will just do bug fixes in fix releases.